lxd/instance/drivers/driver_lxc.go

```diff
@@ -1301,6 +1301,60 @@ func (d *lxc) initLXC(config bool) error {
 	return nil
 }
 
+var idmappedStorageMap map[unix.Fsid]idmap.IdmapStorageType = map[unix.Fsid]idmap.IdmapStorageType{}
+var idmappedStorageMapLock sync.Mutex
+
+// IdmappedStorage determines if the container can use idmapped mounts or shiftfs
+func (d *lxc) IdmappedStorage(path string) idmap.IdmapStorageType {
```

Comment:
@brauner is there a reason you don't just pass in `d.RootfsPath()` every where you call `d.IdmappedStorage("")`?

It looks like the `RootfsPath()` function is already available from the `seccomp` package (which was my first thought as to perhaps why you passed empty string, in case it wasn't available).

Reply:
Pure laziness but sure we can pass it explicitly.

Reply:
@brauner thanks, it helps from a reader's perspective as don't have to drill into the function's implementation to understand what empty path means.